app/src/main/java/pm/gnosis/heimdall/views/TwoStepPanel.kt

```diff
@@ -0,0 +1,210 @@
+package pm.gnosis.heimdall.views
+
+import android.content.Context
+import android.content.res.ColorStateList
+import android.content.res.TypedArray
+import android.graphics.drawable.ColorDrawable
+import android.graphics.drawable.GradientDrawable
+import android.graphics.drawable.ShapeDrawable
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.view.View
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.core.widget.ImageViewCompat
+import com.jakewharton.rxbinding2.view.clicks
+import io.reactivex.Observable
+import kotlinx.android.synthetic.main.layout_two_step_panel.view.*
+import pm.gnosis.heimdall.R
+import pm.gnosis.heimdall.utils.setColorFilterCompat
+import pm.gnosis.svalinn.common.utils.getColorCompat
+
+
+class TwoStepPanel @JvmOverloads constructor(
```

Comment:
Just out of curiosity, why choose a custom layout vs just 2 image views?

Reply:
colouring logic was too cluttery. This way everything is hidden in one place. It makes other layouts that use this panel little bit more short & readable.